espfs/espfs.c

```diff
@@ -40,9 +41,21 @@ It's written for use with httpd, but doesn't need to be used as such.
 #include "espfsformat.h"
 #include "espfs.h"
 
-static char* espFsData = NULL;
+EspFsContext espLinkCtxDef;
+EspFsContext userPageCtxDef;
+
+EspFsContext * espLinkCtx = &espLinkCtxDef;
+EspFsContext * userPageCtx = &userPageCtxDef;
+
+struct EspFsContext
```

Comment:
could you please add some comments about what this struct does and what the fields mean?